ui/app/AppLayouts/Communities/panels/SharedAddressesAccountSelector.qml

```diff
@@ -79,48 +189,113 @@ StatusListView {
                 }
             }
         }
+
+        SortFilterProxyModel {
+            id: accountCollectiblesModel
+
+            sourceModel: walletAccountCollectiblesModel
+
+            filters: ValueFilter {
+                id: filter
+                roleName: "account"
+                value: listItem.accountAddress
+            }
+        }
+
+        ConcatModel {
+            id: concatModel
+
+            sources: [
+                SourceModel {
+                    // During initialization sfpm provides rowCount not taking
+                    // filtering into account, deferring model assignment as
+                    // a workaround
+                    Component.onCompleted: model = walletAccountAssetsModel
```

Comment:
I find it tricky to synchronise this behaviour using `Component.onCompleted`. Looks like filtering runs for sure after `componentComplete`, but it's not guaranteed this `Component.onCompleted` will always be called after SFPM `componentComplete `. Or am I missing something?

I'm thinking it could safer to assign these models on SFPM `Component.onCompleted`. But again, not sure the qml `Component.onCompleted` is called after `componentComplete`.